src/components/hv-text-field/index.js

```diff
@@ -57,12 +59,21 @@ const HvTextField = (props: HvComponentProps) => {
     }
   };
 
+  // Create a memoized, debounced function to trigger the "change" behavior
+  const triggerChangeBehaviors = useCallback(
+    debounce((newElement: Element) => {
```

Comment:
Where is `debounce()` defined?

Reply:
`debounce` is imported from lodash [here](https://github.com/Instawork/hyperview/pull/664/files#diff-72175845930d650efd9d7e8b95f74c4eed08cc08a2a78c67f57b4ed81c2853e2R22)